runtime/mainnet/src/config/collation.rs

```diff
@@ -0,0 +1,288 @@
+use pop_runtime_common::{HOURS, SLOT_DURATION};
+
+use crate::{
+	parameter_types, AccountId, Aura, AuraId, Balances, CollatorSelection, ConstBool, ConstU32,
+	ConstU64, EnsureRoot, PalletId, Runtime, RuntimeEvent, Session, SessionKeys,
+};
+
+impl pallet_authorship::Config for Runtime {
+	type EventHandler = (CollatorSelection,);
+	type FindAuthor = pallet_session::FindAccountFromAuthorIndex<Self, Aura>;
+}
+
+parameter_types! {
+	pub const Period: u32 = 6 * HOURS;
+}
+
+#[docify::export(aura_config)]
+impl pallet_aura::Config for Runtime {
+	type AllowMultipleBlocksPerSlot = ConstBool<true>;
+	type AuthorityId = AuraId;
+	type DisabledValidators = ();
+	// Number of blocks we can produce per session.
+	type MaxAuthorities = Period;
+	type SlotDuration = ConstU64<SLOT_DURATION>;
+}
+
+parameter_types! {
+	pub const PotId: PalletId = PalletId(*b"PotStake");
+	pub const Offset: u32 = 0;
+}
+
+/// We allow root to execute privileged collator selection operations.
+pub type CollatorSelectionUpdateOrigin = EnsureRoot<AccountId>;
+
+impl pallet_collator_selection::Config for Runtime {
+	type Currency = Balances;
+	// should be a multiple of session or things will get inconsistent
+	type KickThreshold = Period;
+	type MaxCandidates = ConstU32<0>;
+	type MaxInvulnerables = ConstU32<20>;
+	type MinEligibleCollators = ConstU32<1>;
```

Comment:
I know we talked about this a bit in DM. For this current mainnet we are preparing, we want the ideal amount for every config as we will have to do one runtime upgrade to get here.

I personally think we should have a minimum of 3 collators (what you originally had). I personally think we should start with 3 collators only as well.

Reply:
I do think 3 would be perfect too!

Let's get the changes into the genesis state first, and then alight this config.

Reply:
I have reverted this change